src/traces/treemap/draw_ancestors.js

```diff
@@ -141,23 +141,31 @@ module.exports = function drawAncestors(gd, cd, entry, slices, opts) {
             s.attr('data-notex', 1);
         });
 
+        var font = Lib.extendFlat({}, helpers.determineTextFont(trace, pt, fullLayout.font, trace.pathdir), {});
```

Comment:
Here too. Shouldn't

```js
var font = helpers.determineTextFont(trace, pt, fullLayout.font, trace.pathdir);
```

be enough?

Reply:
Fixed in https://github.com/plotly/plotly.js/pull/4420/commits/a27dc29ca87303fd53bf994bfe1203b3c5e96887.